hasura/metadata/actions.graphql

```diff
@@ -1,69 +1,107 @@
 type Mutation {
-  adminUpdateUser(payload: AdminUpdateUserInput!): UserResponse
+  adminUpdateUser(
+    payload: AdminUpdateUserInput!
+  ): UserResponse
 }
 
 type Mutation {
-  createCircle(payload: CreateCircleInput!): CreateCircleResponse
+  createCircle(
```

Comment:
how come theres some many blank lines in this file ?

Reply:
**Save action** in Hasura Console is reformatting this file.